docs/guides/genomics-platform/requesting-data/about-our-data.md

```diff
@@ -23,19 +22,38 @@ We provide gVCF files produced by the [Microsoft Genomics Service][msgen]. gVCF
+
+Bambino represents indels in a format incompatible with the VCF specification. When we convert variants to VCFs, we store the original Bambino representation in the `bambino_representation` tag of the info field. Note that this entry is always in HG19 coordinates.
+
+As part of our post-processing pipeline, some variants are annotated with the results of a variety of pathogenicity predictive algorithms. We include those results in the VCFs when they are present. Note that they were performed on the HG19 coordinates. Algorithms include [Likelihood Ratio Test (LRT)][lrt], [MutationAssessor][assessor], [MutationTaster][taster], and [FATHMM][fathmm].
+
+The newly created HG19 VCFs are lifted over using [Picard `LiftoverVcf`][picard]. [`hg19ToHg38.over.chain`][chain] is used as the chain file and `GRCh38_no_alt.fa` is used as the reference. [`bcftools annotate`][bcftools] is used to document the reference file and exact liftover command used in the VCF's header. The HG19 alleles and coordinates are stored by Picard in the `OriginalAlleles`, `OriginalContig`, and `OriginalStart` info tags. There is a bug in the version of Picard used (version 2.18.29) which rarely truncates some of the VCF's genotype fields. These are fields which we use to store read counts and depths for each sequencing type. A [bug report][bug_report] has been filed with Picard. In the current version of the pipeline we recover the dropped entries from the original HG19 VCF. While searching for and correcting these entries, we also reorder the FORMAT and genotype fields to a logical ordering, as opposed to the alphabetical output order of Picard.
 
+`vt normalize` is used to obtain consistent representations of all variants which may have multiple equivalent forms. If VT modifies a variant, it records the original form in the `OLD_VARIANT` info tag. [Click here][vt] to read about variant normalization.
+
+See the [VEP documentation][vep] for a full description of the annotations applied.
+
+!!! note
+    **Variants in these files were curated from analyses across multiple sequencing types including WGS, WES, and RNA Transcriptome.**  
+    For more information on variants for each of the individuals, please refer to the cohort's relevant paper. For more information on the variant calling format (VCF), please see the latest specification for VCF document listed [here][hts-specs].
```

Comment:
I'm also thinking to edit this paragraph to only make mention of 'For more information on the variant calling format (VCF), please see the latest specification for VCF document listed [here][hts-specs].'

Reply:
I agree about taking out the part about "the cohort's relevant paper". I think we've made it clear above this that multiple sequencing types were used. I think we just remove this `note` block, and have a standalone paragraph with the VCF specs sentence.